client/src/store/reducer/mainReducer.js

```diff
@@ -0,0 +1,17 @@
+import { CHECKING_IF_AUTHED } from '../actions/mainActions';
+
+const initalState = {
+  checkingIfAuthed: true,
+};
+
+const adminLoginReducer = (state = initalState, action) => {
```

Comment:
Given that this file exports adminLoginReducer, can the file have the same name?

Reply:
Apologies, adminLoginReducer is wrong. This is a reducer for the component `main`.
Thanks for spotting it!